src/client/src/components/Content/Login/index.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import './styles.css';
+import LoginService from '../../../services/login'
+
+class Login extends Component {
+	constructor(props) {
+		super(props);
+		this.loginService = new LoginService();
+		this.state={
+			username:'',
+			password:'',
+			invalidUsername: false,
+			invalidPassword: false
+		}
+	}
+
+	login(self) {
+
+		if (self.state.username === '') {
+    		self.state.invalidStaffname = true
+    	}
+
+	    if (self.state.password === '') {
+	    	self.state.invalidPassword = true
+	    }
+
+	    if (!self.state.username && !self.state.password) {
+	    	this.loginService.login().then(res => {
+				let login = res.data;
+			});
+	    }
+	}
+
+	render() {
+		return (
+		    <div className="Login">
+				<h1>Login</h1>
+				<input
+					name="username"
+					type = "text"
+					value={this.state.username}
+					placeholder="Enter your Username"
+					onChange = {(event,newValue) => this.setState({username:newValue})}
```

Comment:
Instead of adding the function directly in the render, add it to a function called handleChange and the use that in the render.